### DISCUSSION TITLE
Copy All Metadata Fields for Selected Item

### DISCUSSION THREAD

**Discussion by j3902d** [windingwind/zotero-actions-tags#369] [Q&A]
### Description

Zutilo had a command to copy all of an item's metadata fields to the clipboard as JSON text, including fields that were blank. (See "copy item fields" [here](https://github.com/wshanks/Zutilo/blob/master/docs/COMMANDS.md).) Is there a way to do that with AT?

### Event

None

### Operation

None

### Data

```javascript
/**
 * A description of this script.
 * @author Zotero Community
 * @usage 
 * @link https://github.com/windingwind/zotero-actions-tags/discussions/
 * @see https://github.com/windingwind/zotero-actions-tags/discussions/
 */
```

**Comment by thalient-ai** [comment]
Check out the [Zutilo Copy](https://github.com/windingwind/zotero-actions-tags/discussions/383) and [Zutilo Paste](https://github.com/windingwind/zotero-actions-tags/discussions/384) scripts
